packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -111,5 +113,10 @@ export const messages = {
     'A refresh of your sObject definitions is already underway. If you need to restart the process, cancel the running task.',
   force_project_create_text: 'SFDX: Create Project',
   force_project_create_open_dialog_create_label: 'Create Project',
-  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger'
+  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger',
+
+  isv_debug_bootstrap_step1_create_project:
+    'SFDX: ISV Debugger Setup, Step 1 of 2: Create Project',
```

Comment:
@guw - Is this the command name itself? Or just some placeholder text? 

Could you share a screenshot of it in action?

CC: @ruthemmanuelle

Reply:
I added the screencast to the description!

Reply:
Took me a while to see it. This appears in the output console at the bottom of the screen when the commands are running.